PetaPoco/IAsyncReader.cs

```diff
@@ -17,7 +17,9 @@ public interface IAsyncReader<out T> : IDisposable
         /// <summary>
         /// Asynchronously reads the next row from the data source.
         /// </summary>
-        /// <returns>A task that represents the asynchronous read operation. The task result contains <see langword="true"/> if more records exist; otherwise, <see langword="false"/>.</returns>
+        /// <returns>A task that represents the asynchronous operation. The task result contains <see langword="true"/> if more records exist, otherwise <see langword="false"/>.</returns>
         Task<bool> ReadAsync();
+
+        // TODO: Missing: `Task<bool> ReadAsync(CancellationToken cancellationToken);`
```

Comment:
`IAsyncReader.ReadAsync()` should either accept a `CancellationToken` parameter, or enforce an overload with that parameter. The implementation for `AsyncReader` at https://github.com/CollaboratingPlatypus/PetaPoco/blob/5cd6bdbf208ede225dce461d44666bf6d9240092/PetaPoco/Utilities/AsyncReader.cs#L46 would need updated accordingly also.